kolibri/plugins/learn/assets/src/views/classes/ClassAssignmentsPage.vue

```diff
@@ -43,9 +43,6 @@
         exams: state => state.pageState.currentClassroom.assignments.exams,
         lessons: state => state.pageState.currentClassroom.assignments.lessons,
       }),
-      isMobile() {
-        return this.windowSize.breakpoint <= 1;
-      },
```

Comment:
I've removed most references to breakpoints and standardized on three levels: _small_, _medium_, and _large_ windows. The breakpoints are still available, but typically not necessary to use.